### ISSUE TITLE
Dialog boxes inside same even scrolling instead of popping up

### ISSUE THREAD

**Issue by godai78** [chrismaltby/gb-studio#1005]
**Is your feature request related to a problem? Please describe.**
Change (possible to choose) the way dialog boxes appear. Right now each dialog box (added with a + to a sequence) hides and shows separately (without pause, unlike dialogs in separate events that have a visible pause between).

**Describe the solution you'd like**
Add a different display option, basically like Pokemon/Zelda series, where the dialogs do not hide and pop up, but if the text is extended with + inside and event, the dialog scrolls down shown the next panel.

**Describe alternatives you've considered**
Allow more lines of text and make 3 default and scroll the rest.

**Additional context**

**Comment by pau-tomas** [comment]
> Add any other context or screenshots about the feature request here.

Isn't that already the default behaviour? (I haven't checked in a while)

**Comment by godai78** [comment]
No, it isn't. Adding more text just produces a glitch.

**Comment by godai78** [comment]
Well, this is ALMOST it, but:  
  
a) would be splendid to have it in non-color,  
b) would be nice to be able to chose a number of lines to initially display (or at least choose 3 or 5 - 5 is too much for my liking)  
c) but most importantly what I originally meant was it NOT scrolling on its own, but rather scrolling instead of popping the new window in A keystroke.  
  
And generally c) option is something I would find cool if introduced (even if only in color mode). What it offers now is now what I'm looking for, TBH. At least in present shape with self-scrolling large amount of text.  
  
But thanks for showing me this.

**Comment by darkhog** [comment]
Agreed on being able to set the number of lines before scroll, to any number you want (up to 3/5).For example, in my game I am going for that Pokemon feel, so I want dialogues to be 2 lines high at max.
